data/actionsmap/yunohost.yml

```diff
@@ -1477,6 +1477,39 @@ tools:
 
 
 #############################
+#         Migrations        #
+#############################
+migrations:
+    category_help: Manage migrations
+    actions:
+
+        ### migrations_list()
+        list:
+            action_help: List migrations
+            api: GET /migrations
+
+        ### migrations_migrate()
+        migrate:
```

Comment:
Okay, indeed after thinking about it, if you go 'forward', it wouldnt make much sense to have 'apply'. 'migrate' is okay :+1: